scripts/v1alpha1/create_machine.sh

```diff
@@ -9,10 +10,17 @@ V1ALPHA1_SCRIPTS_PATH="$(dirname "$(readlink -f "${0}")")/"
 USER_DATA_SCRIPT_NAME="user_data.sh"
 SCRIPT_PATH="${V1ALPHA1_SCRIPTS_PATH}""${USER_DATA_SCRIPT_NAME}"
 
+if [[ "${PROVISIONING_IPV6}" == "true" ]]; then
+  PROVISIONING_HOST="[fd2e:6f44:5dd8:b856::1]"
+else
+  PROVISIONING_HOST=172.22.0.1
+fi
+
 MACHINE_NAME=$1
 IMAGE_NAME=${2:-${IMAGE_NAME}}
-IMAGE_URL=http://172.22.0.1/images/${IMAGE_NAME}
-IMAGE_CHECKSUM=http://172.22.0.1/images/${IMAGE_NAME}.md5sum
+
+IMAGE_URL=http://$PROVISIONING_HOST/images/${IMAGE_NAME}
+IMAGE_CHECKSUM=http://$PROVISIONING_HOST/images/${IMAGE_NAME}.md5sum
```

Comment:
Likewise, if we reorder the definitions in lib/common./sh this can be removed I think?

Reply:
The script lets you override the IMAGE_NAME as an argument, so I think this has to stay